cloudformation/media-atom-maker.json

```diff
@@ -345,17 +344,16 @@
                                         ]
                                     ]
                                 },
-                                "aws s3 cp 's3://atom-maker-conf/media-atom-maker.",
+                                "/opt/features/native-packager/install.sh -b atom-maker-dist -a media-atom-maker -t tgz\n",
+
+                                "mkdir /etc/gu \n",
+                                "aws s3 cp 's3://atom-maker-conf/",
                                 {
-                                    "Ref": "Stage"
+                                  "Ref": "Stage"
                                 },
-                                ".application.conf' ",
-                                "/home/media-service/media-atom-maker/conf/application.conf\n",
-                                "cat >> /home/media-service/media-atom-maker/conf/application.conf<<'EOF'\n",
-                                "aws.dynamo.tableName=", { "Ref": "AtomMakerTable" }, "\n",
```

Comment:
Turns out these parameters are used in this cloudformation. I'll do a similarapproach to above but add it to the private conf. In the long run we could put the creation of the dynamo tables in this cloudformation. Then if we want to create a new stack we just have one place to look.